### DISCUSSION TITLE
LMS Tidal app Squeezebox Touch failed to stream

### DISCUSSION THREAD

**Discussion by keyl2021** [LMS-Community/slimserver#691] [Q&A]
Hi, running LMS 8.2.0.
Tidal Premium subscription. Installed Tidal app. Showing in SBT screen, but showing "failed to play stream message". 
Using Tidal Logitech plugin. Other apps like Deezer running fine.
Any clue on how to fix it?

Thanks!!!

**Comment by mherger** [comment]
What OS are you using? What IO::Socket::SSL (see Settings/Information in LMS)? Anything in server.log?

**Reply by keyl2021** [reply]
Using version 7.7.3 r1 16676. Won´t find an available update under "Software Update" option. Thank you!

**Reply by keyl2021** [reply]
Logitech Media Server Version: 8.2.0 - 1627922070 @ Tue Aug 3 11:28:42 WEDT 2021
Hostname: DESKTOP-S7HHLEV
Server IP Address: 192.168.0.27
Server HTTP Port Number: 9000
Operating system: Windows 10 - EN - cp1252
Platform Architecture: 8664
Perl Version: 5.14.1 - MSWin32-x86-multi-thread
Audio::Scan: 1.02
IO::Socket::SSL: 2.068
Database Version: DBD::SQLite 1.58 (sqlite 3.22.0)
Total Players Recognized: 0

**Reply by keyl2021** [reply]
When open LMS application, Player is found. But it is not found when opening LMS from the web.
![SBT1](https://user-images.githubusercontent.com/93448638/139597269-44637de8-7307-447e-b826-a22c7b7c4885.png)

**Reply by mherger** [reply]
Your player is connected to mysqueezebox.com, rather than LMS. You need to connect it to LMS in order to be able to stream TIDAL.

**Reply by keyl2021** [reply]
Thank you Michael! Working fine now.
Would be possible to run Tidal on SBT without computer turned on? Deezer works without LMS turned on on the computer.

Thanks again!
![SBT3](https://user-images.githubusercontent.com/93448638/139719531-10382000-dcd3-4716-afb5-e097f38a9acd.png)

**Reply by michaelherger** [reply]
Unfortunately no, you'll need a recent LMS. But it can easily be run on a cheap Raspberry Pi, which consumes little energy.

**Comment by keyl2021** [comment]
![SBT2](https://user-images.githubusercontent.com/93448638/139597301-2aeea7d2-8bfd-4576-b571-ab1fc312f79d.png)

**Comment by Tecko71** [comment]
My player does not connect to mysqueezebox but is in LMS. Using LMS  8.3.1
